packages/utility/src/_flex.scss

```diff
@@ -106,6 +106,12 @@ $_c: var.$class-flex;
     }
   }
 
+  .#{$_u}#{$_c}-items-fill {
+    > * {
+      flex: 1 0 auto !important;
+    }
+  }
```

Comment:
Add utility for allowing elements inside a flex container to fill the total width available.